.github/CONTRIBUTING.md

```diff
@@ -236,6 +238,24 @@ There are two key points here:
 
 Remember: although this tradeoff makes sense in many cases, it doesn't cover them all. Think carefully about your addition before deciding if you need to use it.
 
+### Prefer `Initialize()` over `New()` for atoms
+Our game controller is pretty good at handling long operations and lag, but it can't control what happens when the map is loaded, which calls `New` for all atoms on the map. If you're creating a new atom, use the `Initialize` proc to do what you would normally do in `New`. This cuts down on the number of proc calls needed when the world is loaded. See here for details on `Initialize`: https://github.com/tgstation/tgstation/blob/master/code/game/atoms.dm#L49
```

Comment:
Probably want to change the link to not link to TG

Reply:
Done. I just removed the link since the link was outdated itself

Reply:
here's the link to our comments on the initialize proc if you want to replace it https://github.com/ParadiseSS13/Paradise/blob/f1de76e50020975e34f1d6039a1047b7e3a24ec8/code/game/atoms.dm#L71

Reply:
I think it's better for them to use Ctrl + T with VSC and then do so